dexbot/basestrategy.py

```diff
@@ -158,7 +158,7 @@ def __init__(
         if config:
             self.config = config
         else:
-            self.config = config = Config.get_worker_config_file(name)
+            self.config = config = Config.get_worker_config_file(name, path=config.config_file)
```

Comment:
Can't reference config here since it's None

Reply:
You're right, just passing config is enough, updated a PR.